web/viewer.js

```diff
@@ -1498,10 +1492,26 @@ function webViewerInitialized() {
     PDFViewerApplication.open(file, 0);
   }
 //#endif
+//#if !PRODUCTION
+  if (true) {
+    return;
+  }
+//#endif
+//#if (FIREFOX || MOZCENTRAL)
+  PDFViewerApplication.setTitleUsingUrl(file);
+  PDFViewerApplication.initPassiveLoading();
+
+  // Clicking on either of the viewBookmark buttons throws a SecurityError
+  // for file:// URLs, hence we hide the buttons since they won't work.
+  if (file && file.lastIndexOf('file:', 0) === 0) {
+    document.getElementById('viewBookmark').classList.add('hidden');
+    document.getElementById('secondaryViewBookmark').classList.add('hidden');
+  }
+//#endif
 //#if CHROME
-//if (file) {
-//  ChromeCom.openPDFFile(file);
-//}
```

Comment:
I'm not sure I really understand your concern here. The viewer still works locally (`node make server`), courtesy of [line 1495](https://github.com/mozilla/pdf.js/pull/6480/files#diff-dd42e9b2d2a652b8e312efa567698aa6R1495) to [line 1499](https://github.com/mozilla/pdf.js/pull/6480/files#diff-dd42e9b2d2a652b8e312efa567698aa6R1499) above.

Note that the build specific code is still enclosed in in preprocessor tags. But I'd like to move away from having blocks of code commented out like this, since that prevents linting, which makes it easier for error to creep in.